cogs5e/initiative/cog.py

```diff
@@ -227,9 +252,24 @@ async def add(self, ctx, modifier: int, name: str, *args):
             init_roll_skeleton = str(init)
 
         me = Combatant.new(
-            name, controller, init, init_skill, hp, ac, private, Resistances.from_dict(resists), ctx, combat
+            name, controller, init, init_skill, hp, ac, private, Resistances.from_dict(resists), ctx, combat, stats
         )
 
+        for skill in profs:
+            skill_key = await search_and_select(ctx, SKILL_NAMES, skill, lambda s: s)
```

Comment:
Since AFAIK the intended user for this interface is programmatic, I would make this a strict check that the given arg was a valid skill key instead (including some documentation).

See the implemention of the Ability Check automation node or `cogs5e.initiative.effects.passive#resolve_check_advs` for examples of this.